scenarios/17_Sneaking_out_of_Raedwood.cfg

```diff
@@ -368,10 +372,178 @@
+        # FIXME: what if Gawen didn't recruit/recall the max amount possible in the previous scenario?
+#ifdef EASY
+        #po: EASY difficulty:
+        {MSGm_Gawen _"I want to avoid fighting. I only have eighteen seasoned troops, and in this neigborhood I can recruit only peasants. I won't be able to find replacements for any of my veterans, and this is just the beginning of our quest."}
+#endif
+#ifdef NORMAL
+        #po: NORMAL difficulty:
+        {MSGm_Gawen _"I want to avoid fighting. I only have fifteen seasoned troops, and in this neigborhood I can recruit only peasants. I won't be able to find replacements for any of my veterans, and this is just the beginning of our quest."}
+#endif
+#ifdef HARD
+        #po: HARD difficulty:
         {MSGm_Gawen _"I want to avoid fighting. I only have twelve seasoned troops, and in this neigborhood I can recruit only peasants. I won't be able to find replacements for any of my veterans, and this is just the beginning of our quest."}
-        {MSG_Ruvio _"If so, then it is up to you to figure out a strategy which will avoid any losses. While will have to pass near the guards, and may skirmish with them, there is no need to get into a stand-up fight with them."}
+#endif
+#ifdef NIGHTMARE
+        #po: unused:
+        {MSGm_Gawen _"I want to avoid fighting. I barely have any seasoned troops, and in this neigborhood I can recruit only peasants. I won't be able to find replacements for any of my veterans, and this is just the beginning of our quest."}
+#endif
+        {MSG_Ruvio _"If so, then it is up to you to figure out a strategy which will avoid any losses. While we will have to pass near the guards, and may skirmish with them, there is no need to get into a stand-up fight with them."}
         {MSGm_Lorin _"How typical for an underling, trying to avoid conflict. My son, let's just slaughter them all! More enemies, more honor!"}
         {MSG_Ruvio _"With that kind of attitude, how have you managed to survive this long? We'll want to manage our resources, too - my guess is that we should have at least 250 golden coins at the end of this battle."}
+        {MSGm_Gawen _"Hm, there are certainly many decisions to be made here..."}
+        {IF_HAVE_UNIT_ANY (Reme Carrenemoe)}
+        {MSGm_Reme _"Well, Gawen, which path will you take? The stealthy route proposed by Ruvio, or the violent route proposed by your stepmother?"}
+        {ELSE}
+        {IF_HAVE_UNIT_ANY (Reumario)}
+        {MSG_Reumario _"Well, Gawen, which path will you take? The stealthy route proposed by the underling, or the violent route proposed by the 'She-Wolf of Haeltin'?"}
+        {ELSE}
+        {MSG_Ruvio _"Well, Gawen, which path will you take? The stealthy route I proposed, or the violent route proposed by your stepmother?"}
+        {TWO_END_IFs}
+        {MSGOPTI3 (Gawen Hagarthen) (portraits/gawen_was.png) (_"Well...")
+        (_"I think I'll take the stealthy strategy") (stealth)
```

Comment:
It's interesting to have these, I haven't playtested to see how they actually feel though. Not sure what you think. Is it actually an improvement?

Reply:
The stealthy path is definitely an improvement IMO; it makes it much easier to actually put the "Sneaking" in the scenario's title. The violent path is kind of trickier, since having berserk makes it much easier to die, but I could see players with a certain playstyle appreciating it. That's why it's a choice, after all.